# Portuguese translation of liveusb-creator.
# Copyright (C) 2008  Red Hat, Inc.
# This file is distributed under the same license as the liveusb-creator package.
# Valter Fukuoka <dotsoft@gmail.com>, 2008.
#
msgid ""
msgstr ""
"Project-Id-Version: liveusb-creator 2.7\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2008-08-27 09:34-0400\n"
"PO-Revision-Date: 2008-08-28 21:12-0800\n"
"Last-Translator: Valter Fukuoka <dotsoft@gmail.com>\n"
"Language-Team: PT <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: ../liveusb/gui.py:69
#, python-format
msgid "Unknown release: %s"
msgstr "Edição (versão) desconhecida: %s"

#: ../liveusb/gui.py:73
#, python-format
msgid "Downloading %s..."
msgstr "Descarregando %s..."

#: ../liveusb/gui.py:152
msgid "Verifying filesystem..."
msgstr "Verificando o sistema de arquivos..."

#: ../liveusb/gui.py:155
msgid "Error: Cannot set the label or obtain the UUID of your device.  Unable to continue."
msgstr "Erro: Impossível setar o nome(label) ou obter o UUID do dispositivo. Não é possível continuar."

#: ../liveusb/gui.py:164
msgid "Verifying SHA1 of LiveCD image..."
msgstr "Verificando o SHA1 da imagem LiveCD..."

#: ../liveusb/gui.py:166
msgid "Error: The SHA1 of your Live CD is invalid.  You can run this program with the --noverify argument to bypass this verification check."
msgstr "Erro: O SHA1 do Live CD está incorreto. Você pode executar esse programa com o argumento --noverify para evitar que essa verificação ocorra."

#: ../liveusb/gui.py:177
msgid "Extracting live image to USB device..."
msgstr "Extraindo a imagem Live para o dispositivo USB..."

#: ../liveusb/gui.py:180
#, python-format
msgid "Creating %d Mb persistent overlay..."
msgstr "Criando %d Mb para overlay persistente..."

#: ../liveusb/gui.py:183
msgid "Configuring and installing bootloader..."
msgstr "Configurando e Instalando o bootloader (carregador de inicialização)"

#: ../liveusb/gui.py:187
#, python-format
msgid "Complete! (%s)"
msgstr "Completo! (%s)"

#: ../liveusb/gui.py:190
#: ../liveusb/gui.py:193
msgid "LiveUSB creation failed!"
msgstr "Criação de LiveUSB falhou!"

#: ../liveusb/gui.py:316
#, python-format
msgid "Persistent Storage (%d Mb)"
msgstr "Memória Persistente (%d Mb)"

#: ../liveusb/gui.py:334
msgid ""
"Your device already contains a LiveOS.\n"
"If you continue, this will be overwritten."
msgstr ""
"O seu dispositivo já contém um LiveOS.\n"
"Caso continue, ele será re-escrito (escrito em cima)."

#: ../liveusb/gui.py:337
msgid "Warning: Creating a new persistent overlay will delete your existing one."
msgstr "Advertência: Criar um novo overlay persistente irá apagar o que já existe."

#: ../liveusb/gui.py:339
msgid "Press 'Create Live USB' again if you wish to continue."
msgstr "Pressione 'Criar Live USB' novamente caso deseje continuar."

#. The user has confirmed that they wish to overwrite their
#. existing Live OS.  Here we delete it first, in order to
#. accurately calculate progress.
#: ../liveusb/gui.py:349
msgid "Removing existing Live OS..."
msgstr "Removendo o Live OS (sistema) existente..."

#: ../liveusb/gui.py:383
msgid "Download complete!"
msgstr "Descarga completa!"

#: ../liveusb/gui.py:387
msgid "Download failed: "
msgstr "Descarga falhou:"

#: ../liveusb/gui.py:388
msgid "You can try again to resume your download"
msgstr "Tente retornar ao download(descarga) novamente"

#: ../liveusb/gui.py:392
msgid "Select Live ISO"
msgstr "Selecione Live ISO"

#: ../liveusb/gui.py:399
msgid "Sorry, I'm having trouble encoding the filename of your livecd.  You may have better luck if you move your ISO to the root of your drive (ie: C:\\)"
msgstr "Infelizmente existe problema na codificação do nome de seu livecd. Tente mover a imagem ISO para a raíz do drive (ie: C:\\)"

#: ../liveusb/gui.py:404
#, python-format
msgid "ISO selected: %s"
msgstr "ISO selecionado: %s"

#: ../liveusb/creator.py:146
#, python-format
msgid ""
"There was a problem executing the following command: `%s`\n"
"A more detailed error log has been written to 'liveusb-creator.log'"
msgstr ""
"Problema encontrado ao executar o seguinte comando: `%s`\n"
"Um arquivo log detalhado foi escrito em 'liveusb-creator.log'"

#: ../liveusb/creator.py:183
msgid "Not enough free space on device."
msgstr "Não existe espaço suficiente no dispositivo."

#: ../liveusb/creator.py:190
#, python-format
msgid "Creating %sMB persistent overlay"
msgstr "Criando %sMB para overlay persistente"

#: ../liveusb/creator.py:226
#, python-format
msgid "Unable to remove previous LiveOS: %s"
msgstr "Não foi possível remover o LiveOS (sistema) existente anteriormente: %s"

#: ../liveusb/creator.py:257
#, python-format
msgid "Cannot find device %s"
msgstr "Não foi possíve localizar o dispositivo %s"

#: ../liveusb/creator.py:306
msgid "Unable to find any USB drives"
msgstr "Não foi possível localizar discos USB"

#: ../liveusb/creator.py:327
#, python-format
msgid "Filesystem for %s unknown!"
msgstr "Sistema de arquivos para %s desconhecido!"

#: ../liveusb/creator.py:335
#, python-format
msgid "Unable to mount device: %s"
msgstr "Não foi possível montar o dispositivo: %s"

#: ../liveusb/creator.py:370
#, python-format
msgid "Unknown filesystem for %s.  Your device may need to be reformatted."
msgstr "Sistema de arquivos desconhecido %s. O seu dispositivo talvez necessite ser reformatado."

#: ../liveusb/creator.py:373
#, python-format
msgid "Unsupported filesystem: %s"
msgstr "Sistema de arquivos não suportado: %s"

#: ../liveusb/creator.py:400
msgid "Unable to find LiveOS on ISO"
msgstr "Não foi possível localizar um LiveOS (sistema) no ISO"

#: ../liveusb/creator.py:472
msgid "Unable to find any removable devices"
msgstr "Não foi possível localizar dispositivos de disco removíveis"

#: ../liveusb/creator.py:479
msgid "Make sure your USB key is plugged in and formatted with the FAT filesystem"
msgstr "Certifique-se de que a chave USB está conectada e formatada com sistema de arquivos FAT"

#: ../liveusb/creator.py:482
#, python-format
msgid ""
"Unsupported filesystem: %s\n"
"Please backup and format your USB key with the FAT filesystem."
msgstr ""
"Sistema de arquivos não suportado: %s\n"
"Por favor, faça um backup e formate seu disco USB com sistema de arquivos FAT."

#: ../liveusb/creator.py:558
#, python-format
msgid "Cannot find '%s'.  Make sure to extract the entire liveusb-creator zip file before running this program."
msgstr "'%s' não encontrado. Certifique-se de extrair o arquivo liveusb-creator zip inteiramente antes de executar esse programa."

